csvqb/csvqb/tests/unit/test_baseunit.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from abc import ABC
+from typing import List
+import json
+from pathlib import Path
+
+
+def get_test_base_dir() -> Path:
+    path_parts = Path(".").absolute().parts
+    test_index = path_parts.index("tests")
+    test_root_path = Path(*path_parts[0 : test_index + 1])
+    return test_root_path
```

Comment:
Didn't you talk about wanting to do some kind of improvement here? I wrote a mad function to find the `test-cases` directory somewhere in one of the behave files.

Reply:
https://github.com/GSS-Cogs/csvwlib/pull/103/files#diff-5e98b084209290860da165f7a1746943e7ee019ae42045cdeded1126f55f86fdR33